modules/fx/arrow-fx-rx2/src/main/kotlin/arrow/fx/rx2/FlowableK.kt

```diff
@@ -24,17 +24,17 @@ class ForFlowableK private constructor() {
   companion object
 }
 typealias FlowableKOf<A> = arrow.Kind<ForFlowableK, A>
-typealias FlowableKKindedJ<A> = io.kindedj.Hk<ForFlowableK, A>
```

Comment:
Are we removing KindedJ support? @pakoito

Reply:
No, covariance needs a compatibility function that's on the `arrow-kindedj` module.